.github/workflows/build_and_test.yml

```diff
@@ -600,19 +600,16 @@ jobs:
         done
     - name: Install Conda for pip packaging test
       if: contains(matrix.modules, 'pyspark-errors')
-      run: |
-        curl -s -L "https://github.com/conda-forge/miniforge/releases/download/24.11.2-1/Miniforge3-Linux-x86_64.sh" > miniforge3.sh
-        bash miniforge3.sh -b -p $HOME/miniforge3
-        rm miniforge3.sh
+      uses: conda-incubator/setup-miniconda@v3
```

Comment:
Are we sure that this is ASF-approved Action? I would be great if you can provide the example of another ASF project using this (in your PR description). Is this is not approved one, it will fail after merging.

Reply:
Thanks for your reminding!
- It seems that `Apache Arrow` also used this action, as follows:
https://github.com/apache/arrow/blob/5109c0c485d5dc53c83d189a2f4a27119708a957/dev/tasks/verify-rc/github.win.yml#L37
<img width="611" alt="image" src="https://github.com/user-attachments/assets/5654deec-7109-4e6b-bee5-280aa642cad8" />

- https://github.com/apache/infrastructure-actions/blob/cacf961f13a58f4f0e178e7663e11394b88581e8/approved_patterns.yml#L14
<img width="575" alt="image" src="https://github.com/user-attachments/assets/9da3b7c6-d598-4ee8-b777-9dda477e45c0" />

`apache/infrastructure-actions` also allows `conda-incubator/setup-miniconda@*`

Reply:
Nice! Please copy the above comments into the PR description, @panbingkun !

Reply:
Done.